pom.xml

```diff
@@ -2325,6 +2326,25 @@
       </build>
     </profile>
     <profile>
+      <id>eclipse</id>
```

Comment:
please rename it to "eclipse-compile".

please add it usage to .travis.yml, as integration is easy here:
```
$ git diff
diff --git a/.travis.yml b/.travis.yml
index d6c5a12..df21c0d 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -58,6 +58,12 @@ matrix:
         - DESC="findbugs, spotbugs and pmd"
         - CMD="mvn clean compile pmd:check findbugs:check spotbugs:check"
 
+    # findbugs, spotbugs and pmd (oraclejdk8)
+    - jdk: oraclejdk8
+      env:
+        - DESC="eclipse static analysis"
+        - CMD="mvn clean compile -Peclipse-compile"
+
     # Releasenotes generation - validaton
     - jdk: oraclejdk8
       env:
```